source/keyboardHandler.py

```diff
@@ -567,7 +574,11 @@ def executeScript(self, script):
 		# Now actually execute the script.
 		super().executeScript(script)
 
+	#: The maximum amount of time (in ms) to wait for keys injected by NVDA to be
+	#: received by NVDA.
+	_INJECTION_WAIT_TIMEOUT: int = 10

```

Comment:
Added new line for lint

```suggestion
	_INJECTION_WAIT_TIMEOUT: int = 10

```